status-updates/2023/2023-08-17.md

```diff
@@ -0,0 +1,223 @@
+# Thursday, August 17, 2023, 11:00am Ro
+
+## Agenda
+
+- Skeleton Contents milestone (lead: RăzvanN)
+- Difficulties in porting content (lead: Gabi)
+- M4 Event 13 September 2023 (lead: RazvanD)
+
+## Participants
+
+RăzvanD, RăzvanN, Ștefan, Teo, Alex, Sergiu, Gabi, Adrian
+
+## Discussions
+
+### Skeleton Contents
+
+RD: There is little cohesion between the components that make up the methodology.
+
+RD: The methodology must answer the following questions:
+
+- How can I develop / create a digital educational content repository? 
+- How do I use (as an educator) a digital educational content repository as part of an educational implementation (course, training, workshop, etc.)?
+- How do I contribute to a digital educational content repository?
+- How do I maintain a digital educational content repository?
+
+RD: We are focusing on how to use the resources and are targeting the developer, teacher, contributor and manager / curator/
+Administrating the infrastructure is something unrelated.
+We need to assign roles per section so any of the above users knows where to look.
+
+RD: We are proposing the following repository structure:
+
+- A content repository that is organised and used as described in the "Developing and Organizing Content" section.
```

Comment:
```suggestion
- A content repository that is organized and used as described in the "Developing and Organizing Content" section.
```
zz

Reply:
Yes, I am aware, I was just pointing out your inconsistency. Pick one and reformat.